src/components/ContentHighlights/HighlightStepper/ContentHighlightStepper.jsx

```diff
@@ -84,9 +84,9 @@ const ContentHighlightStepper = ({ enterpriseId }) => {
         highlightedContentUuids: result.highlightedContent || [],
       };
       dispatchEnterpriseCuration(enterpriseCurationActions.addHighlightSet(transformedHighlightSet));
+      dispatchEnterpriseCuration(enterpriseCurationActions.setHighlightToast(transformedHighlightSet.uuid));
```

Comment:
May want to ensure this line gets test coverage (i.e., was `setHighlightToast` called with the expected `uuid`)?

Reply:
Deferring as a separate chunk of work since `handlePublish` doesn't currently have any coverage anyways.